jupyter-docker/build.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+build() {
+    echo "building jupyter docker image..."
+    docker build -t broadinstitute/leonardo-notebooks:gawb2624_2 .
+}
+
+push() {
+    echo "pushing jupyter docker image..."
+    docker push broadinstitute/leonardo-notebooks:gawb2624_2
+}
+
+JUPYTER_COMMAND=$1
+
+JUPYTER_REPO=$2
```

Comment:
This is not used.  To agree with the readme, it should be the tag, and should be used by `build` and `push`